codec-http2/src/main/java/io/netty/handler/codec/http2/Http2ConnectionHandler.java

```diff
@@ -734,10 +734,10 @@ protected void onStreamError(ChannelHandlerContext ctx, boolean outbound,
 
         if (stream == null) {
             if (!outbound || connection().local().mayHaveCreatedStream(streamId)) {
-                resetUnknownStream(ctx, streamId, http2Ex.error().code(), ctx.newPromise());
```

Comment:
This method will be called later then via the `resetStream(...)` method that is part of the `Http2LifecycleManager` interface